index.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const knex = require('knex')({
```

Comment:
storing credentials in the code is a no go :) Please extract them into environment variables.